app/main.py

```diff
@@ -1,3 +1,17 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: any) -> any:
+    cached_list = {}
+
+    def inner(*args) -> any:
+        if args in cached_list:
+
+            cached_list[args] = "Getting from cache"
+            print("Getting from cache")
+
+            for item in args:
+                return item
+
+        cached_list[args] = "Calculating new result"
```

Comment:
you should store the result of the function not a message

Reply:
You wrote earlier: sue dictionary as cache, and store args as key, and result of the func as value. I store result and message. Its not correct?

Reply:
no store the args as key, and result of function as value